web/cypress/integration/route/create-edit-delete-route.spec.js

```diff
@@ -72,12 +73,25 @@ context('Create and Delete Route', () => {
     });
 
     // config prometheus plugin
-    cy.contains(this.domSelector.pluginCard, 'prometheus').within(() => {
-      cy.get('button').first().click({
-        force: true
-      });
+    cy.contains('prometheus').parents('.ant-card-bordered').within(() => {
+      cy.get('button').click({ force: true });
+    });
+
+    cy.get('.ant-drawer-content').should('be.visible').within(() => {
+      cy.get('#disable').click();
+      cy.get('.ant-switch-checked').should('exist');
+    });
+
+    cy.contains('button', 'Submit').click();
+    cy.get('.ant-drawer-content', { timeout }).should('not.exist');
+
+    cy.contains('prometheus').parents('.ant-card-bordered').within(() => {
+      cy.get('button').click({ force: true });
     });
-    cy.contains('button', 'Cancel').click();
+
+    cy.get('.ant-drawer-footer').contains('button', 'Delete').click({ force: true });
+    cy.contains('button', 'Confirm').click({ force: true });
+
```

Comment:
Also use `this.domSelector..` and `this.data...`
I think another step should be added: check that the plugin was successfully removed after it was deleted.